lumicks/pylake/group.py

```diff
@@ -21,3 +22,25 @@ def __getitem__(self, item):
         else:
             cls = channel_class(thing)
             return cls.from_dataset(thing)
+
+    def keys(self):
+        """Return group names at this level"""
+        return list(self.h5.keys())
+
+    def __iter__(self):
+        self._idx = -1
+        return self
+
+    def __next__(self):
+        """Return key names for the iterator analogously to HDF5"""
+        if self._idx >= len(self.keys()) - 1:
+            raise StopIteration
+
+        self._idx += 1
+        return self.keys()[self._idx]
+
+    def __repr__(self):
+        """Return type name and members of the group"""
+        name = self.__class__.__name__
+        members = ''.join(f"{e}, " for e in self.keys())
+        return f"{name} (members: {members[:-2]})"
```

Comment:
If we're displaying 'keys' we should not call them 'members'. We should also consider the most idiomatic way to display this data. If we consider that a group is like a `dict`, then `{'key': 'value', ...}` would make sense. `value` is a bit problematic because it might be too much information, so maybe just `{'key1', 'key2', ...}`. That's more like a `set` than a `dict` but at least we're sticking closer to the language idioms.

Reply:
Alright. {'key1', 'key2', ...} it is.